docs/data/system/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -95,7 +95,7 @@ If you have custom breakpoints, the change is the same:
 Which you can cover with the same codemod by providing the custom breakpoints as an argument:
 
 ```bash
-npx @mui/codemod@next v6.0.0/grid-v2-props <path/to/folder> --jscodeshit='--muiBreakpoints=mobile,desktop'
+npx @mui/codemod@next v6.0.0/grid-v2-props <path/to/folder> --jscodeshift='--muiBreakpoints=mobile,desktop'
```

Comment:
@mnajdova the same typo exist in `docs/data/material/migration/migrating-to-v6/migrating-to-v6.md`

Reply:
I thought I changed it, but it wasn't saved :), pushed.